web3/providers/ipc.py

```diff
@@ -91,21 +91,15 @@ def get_default_ipc_path() -> Optional[str]:
         ipc_path = os.path.expanduser(
```

Comment:
Nit: Since these `ipc_path` vars are not used anywhere else, just return in a single line.
```suggestion
        return os.path.expanduser(
```